sources/packages/api/src/main.ts

```diff
@@ -6,6 +6,11 @@ async function bootstrap() {
   const app = await NestFactory.create<NestExpressApplication>(AppModule);
   app.setGlobalPrefix('api')
   const port = process.env.PORT || 3000;
```

Comment:
Is there any merit in externalizing the default port ?

Reply:
@2011sandeepraj, this is not part of the changes in this PR. Maybe we can start a conversation about it outside this review.